### ISSUE TITLE
Exhaustion cannot be increased or decreased on the actor sheet, only toggled

### ISSUE THREAD

**Issue by krbz999** [foundryvtt/dnd5e#2857]
Seems this functionality doesn't carry over from the token hud.

**Comment by arbron** [comment]
Well it can be using the exhaustion pips, but I imagine this is asking about the toggle in the conditions list. Not sure that would be the best, since the design of the conditions area is clearly Toggles, so adding extra behavior here would go against the visual design and be potentially confusion.

**Comment by Fyorl** [comment]
Yeah, exhaustion is in a bit of an awkward spot on the effects tab currently due to it not being binary like the other conditions. Not sure what to do with it at the moment.

**Comment by YenBenGrey** [comment]
Just to add to this, the effects of turning Exhaustion off in the conditions section of the character sheet means that you cannot turn it back on with the pips or token HUD, until, and this is the bizarre part, you go in to the token HUD then right click Exhaustion icon and then left click Exhaustion icon OR you go to the pips and choose anything higher than 1 and then reduce to one.  Those methods kind of unlocks the fault. 

This causes a problem with modules and macros that increase the Exhaustion too from a zero Exhaustion situation.

**Comment by Sayshal** [comment]
Resolved with #6394
